src/managers/config.py

```diff
@@ -186,6 +189,9 @@ def zookeeper_properties(self) -> list[str]:
             + self.metrics_exporter_config
         )
 
+        if self.config.enforce_sasl_client:
+            properties.extend(CLIENT_SASL_AUTH.splitlines())
+
```

Comment:
**huge nitpicky suggestion:** Could always do:

```python
+ self.metrics_exporter_config
+ CLIENT_SASL_AUTH.splitline() if self.config.enforce_sasl_client else []
```

Reply:
Your wish is my command:  d135227be7